python/lsst/obs/decam/decamMapper.py

```diff
@@ -145,6 +145,18 @@ def bypass_deepMergedCoaddId(self, datasetType, pythonType, location, dataId):
     def bypass_deepMergedCoaddId_bits(self, *args, **kwargs):
         return 64 - DecamMapper._nbit_id
 
+    def bypass_dcrCoaddId(self, datasetType, pythonType, location, dataId):
+        return self._computeCoaddExposureId(dataId, True)
+
+    def bypass_dcrCoaddId_bits(self, *args, **kwargs):
```

Comment:
Same question as in lsstSim: should the bypass bits calls just call the equivalent coadd version, or are they distinct (but currently identical) for a reason?